playbooks/aws/ansible-tower/launch.yml

```diff
@@ -0,0 +1,103 @@
+---
+- name: Launch instance(s)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+
+  vars:
+    inst_region: us-east-1
+    rhel7_ami: ami-a24e30ca
+    user_data_file: user_data.txt
+    oo_vpc_subnet_id:    # Purposely left blank, these are here to be overridden in env vars_files
+    oo_assign_public_ip: # Purposely left blank, these are here to be overridden in env vars_files
```

Comment:
I don't think we'll ever launch a tower instance outside of a VPC, so let's remove all of the code that would allow it to be launched in classic. These two vars aren't needed here, nuke them.